packages/arb-avm-cpp/avm/src/machinestate/machinestate.cpp

```diff
@@ -124,6 +124,22 @@ uint256_t MachineState::hash() const {
     return from_big_endian(hashData.begin(), hashData.end());
 }
 
+bool MachineState::verifyMachineValidity() {
+    int machine_size = 0;
+
+    machine_size += code.size();
```

Comment:
This doesn't really work since the code vector isn't really part of the machine the way it's hashed. Instead we just have the instruction stack. We should either just use the size of the code as 1 since it's a codepoint, or we could just ignore it entirely